olad/Strerror_r.cpp

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I wonder if these should be in include/ola and common/ola for future use by other bits of the codebase.

Reply:
Could do that, but I'd have to add `strerror_r` to the main list of functions checked. Probably not a problem?

Reply:
I can't remember, are they mandatory functions, or still optional?

Reply:
Well `AC_CHECK_FUNCS` wouldn't terminate if something is not found, so it's alright.